src/jobtypes.py

```diff
@@ -68,11 +67,27 @@ def __init__(self, repo_name='mozilla-inbound'):
                 #  opt test web-platform-tests-4"
                 if job['build_system_type'] == 'buildbot':
                     testtype = job['ref_data_name'].split(' ')[-1]
+                    job_identifier = job['ref_data_name']
 
                 # taskcluster's testtype is a part of its 'job_type_name' like reftest-2
                 # for [TC] Linux64 reftest-2
                 else:
-                    testtype = job['job_type_name'].split(' ')[-1]
+                    # we don't want job of build kind in seta
+                    if job['job_type_symbol'] == "I":
+                        continue
+                    else:
```

Comment:
If you want, you could whitelist test jobs instead of blacklisting build jobs.

For now, you can check that the label starts with 'desktop-test'.
In the future, we would have to add other platforms.